docs/docs/APIGuide/MLPipeline/DLEstimator/DLEstimator.md

```diff
@@ -0,0 +1,84 @@
+## DLEstimator ##
+
+**Scala:**
+
+```scala
+val estimator = new DLEstimator(model: Module[T], criterion: Criterion[T], val featureSize: Array[Int], val labelSize: Array[Int])
+```
+
+Source code [here](https://github.com/intel-analytics/BigDL/blob/585818a3fb0e7339eb4e3831f08da82b7d5e47ba/spark/dl/src/main/scala/org/apache/spark/ml/DLEstimator.scala#L53).
+
+In machine learning context, "Estimation" is regarded as a mechanism for choosing the relatively "best model" according to the observation of reality. The API design of the `Estimator` abstracts the concept of a learning algorithm or any algorithm that fits or trains on data to make the training pipeline more clear and convinient for our users. Technically, an Estimator takes the user-presumed model and criterion with specification on feature and label dimension to prepare for the training. Within its class definition, it implements a method `fit()`, which accepts dataset to start training and produce a optimized DLModel with more accurate prediction.

```

Comment:
Merge the above paragraph with the following one. Maybe for explaining the `Estimator`, it's better if we just add a link to the Spark doc of Estimator here.